docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -0,0 +1,417 @@
+---
+id: configure-hooks
+title: Add and configure hooks for self-service user flows
+sidebar_label: Configuration
```

Comment:
What about "Flow customization"?

Reply:
As mentioned in the previous comment, being unambiguous is very important in docs. In "Flow customization" I feel like "flow" can mean many different things and so can "customization".